api/base.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+import "google/protobuf/timestamp.proto";
+option go_package = "internal/proto/base";
+
+package base;
+
+message DebugData {
```

Comment:
This is not just for debugging IMHO -- `RequestInfo` might be a better name

Reply:
Sure, what is the idea behind the RequestInfo designation?

Reply:
where else would the data be used?

Reply:
Request ids and timestamps are used for general monitoring and analysis of distributed systems. We might e.g. be interested in performance aspects which are not related to debugging.

Reply:
change made in 050a2ab